.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
eslint is alr in the repo, make sure ur stuff is compatible, or change all the way to this one